### DISCUSSION TITLE
Are there any plans to add a video player?

### DISCUSSION THREAD

**Discussion by learningchannel** [datenstrom/community#203] [Ask a question] [answered]
http://docs.afterglowplayer.com

anyone care to make a plugin for AfterGlow -- a self hosted video player? it would be useful to use this much like [gallery] or [fotorama] as part of the basic package. thanks in advance.

**Comment by nibreh** [comment]
Hi ! 

You can use [this plugin based on videojs](https://github.com/varakh/yellowcms-extensions-videojs) by @varakh  - but maybe it's not updated to yellow v0.6.5 ?

It's easy to make a plugin by reading the source code of another one and change few lines. Check the [audio tag plugin](https://github.com/schulle4u/yellow-plugin-audio) for inspiration.

If  I have time, I can make this plugin. Be patient ;)

**Comment by nibreh** [comment, marked as answer]
I made the plugin for HTML5 video player: https://github.com/nibreh/yellow-plugin-video

It use the HTML5 default player with `<video>` tag.  Should be supported by most modern browsers.
Based on [HTML5 audio plugin](https://github.com/schulle4u/yellow-plugin-audio) by @schulle4u

I prefer to use the defaults HTML5 tags because it's the standard way to show video on the web, simple & functional. Others scripts like _afterglowplayer.js_ or _video.js_ are for cosmetic design & various options. 

If you want to use _afterglow.js_, just modify the video plugin like you want ! I can help you of course.

**Comment by markseuffert** [comment]
Good work, thank you @Nibreh.

**Comment by learningchannel** [comment]
thank you!
